Framework/DataHandling/src/GenerateGroupingPowder.cpp

```diff
@@ -244,7 +239,7 @@ void GenerateGroupingPowder::init() {
   fileExtensionXMLorHDF5->addAllowedValue(std::string("nxs"));
   fileExtensionXMLorHDF5->addAllowedValue(std::string("nx5"));
   declareProperty("FileFormat", std::string("xml"), fileExtensionXMLorHDF5,
-                  "File extension/format for saving output: either xml (default) or nxs/nx5.", PropertyMode::Optional);
+                  "File extension/format for saving output: either xml (default) or nxs/nx5.");
```

Comment:
It should be optional.  The original work was to allow this to output just a grouping workspace instead, for use in SNAPRed, without more loading.